### PR TITLE
fix go generate

### PR DESCRIPTION
**Pull request by jrcamp**
go generate would not work because a tool (esc) was missing from install-tools.

fixing in preparation for metrics metadata generation.

### CODE REVIEW

File: service/internal/resources.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright  OpenTelemetry Authors
+// Copyright 2020 The OpenTelemetry Authors
```
Comment (bogdandrutu) [inline service/internal/resources.go:1]: Should we skip the year?

Comment (jrcamp) [reply service/internal/resources.go:1]: @bogdandrutu currently addlicense in Makefile adds the year so I was just trying to maintain consistency.

Comment (bogdandrutu) [reply service/internal/resources.go:1]: I think we removed globally

Comment (james-bebbington) [reply service/internal/resources.go:1]: https://github.com/google/addlicense/issues/36 😆

Comment (jrcamp) [reply service/internal/resources.go:1]: > I think we removed globally

@bogdandrutu I'm confused, it's still in a bunch of the file headers so I don't think it's been removed?

Comment (jrcamp) [reply service/internal/resources.go:1]: Do you want me to add -y"" here and in Makefile?

Comment (bogdandrutu) [reply service/internal/resources.go:1]: Please do in a separate PR :)

**Comment by bogdandrutu** [comment]
Just rebase this :)

**Comment by jrcamp** [comment]
@bogdandrutu done

**Pull request merged**